l1-contracts/src/core/Rollup.sol

```diff
@@ -311,6 +321,10 @@ contract Rollup is EIP712("Aztec Rollup", "1"), Leonidas, IRollup, ITestRollup {
       }
     }
 
+    for (uint256 i = 0; i < _epochSize; i++) {
+      // free up gas (hopefully)
```

Comment:
Unless you are submitting the proof in the same transaction as you set the values originally you will be paying extra to do this. Cleaning it here is essentially doing like gas-tokens did back in the days before pricing was changed. 
Would remove.